src/server/scripts/Commands/cs_reset.cpp

```diff
@@ -296,6 +309,418 @@ class reset_commandscript : public CommandScript
+        }
+        else
+        {
+            int16 deletedItemsCount = ResetItemsInBags(targetPlayer);
+            handler->PSendSysMessage(LANG_COMMAND_RESET_ITEMS_BAGS, deletedItemsCount, handler->GetNameLink(targetPlayer).c_str());
+        }
+
+        return true;
+    }
+
+    static bool HandleResetItemsKeyringCommand(ChatHandler* handler, Optional<PlayerIdentifier> target)
+    {
+        Player* targetPlayer = GetPlayerFromIdentifierOrSelectedTarget(handler, target);
+
+        if (!targetPlayer)
+        {
+            return false;
+        }
+        else
+        {
+            int16 deletedItemsCount = ResetItemsInKeyring(targetPlayer);
+            handler->PSendSysMessage(LANG_COMMAND_RESET_ITEMS_KEYRING, deletedItemsCount, handler->GetNameLink(targetPlayer).c_str());
+        }
+
+        return true;
+    }
+
+    static bool HandleResetItemsInCurrenciesListCommand(ChatHandler* handler, Optional<PlayerIdentifier> target)
+    {
+        Player* targetPlayer = GetPlayerFromIdentifierOrSelectedTarget(handler, target);
+
+        if (!targetPlayer)
+        {
+            return false;
+        }
+        else
+        {
+            int16 deletedItemsCount = ResetItemsInCurrenciesList(targetPlayer);
+            handler->PSendSysMessage(LANG_COMMAND_RESET_ITEMS_CURRENCY, deletedItemsCount, handler->GetNameLink(targetPlayer).c_str());
+        }
+
+        return true;
+    }
+
+    static bool HandleResetItemsInBankCommand(ChatHandler* handler, Optional<PlayerIdentifier> target)
+    {
+        Player* targetPlayer = GetPlayerFromIdentifierOrSelectedTarget(handler, target);
+
+        if (!targetPlayer)
+        {
+            return false;
+        }
+        else
+        {
+            int16 deletedItemsCount = ResetItemsInBank(targetPlayer);
+            handler->PSendSysMessage(LANG_COMMAND_RESET_ITEMS_BANK, deletedItemsCount, handler->GetNameLink(targetPlayer).c_str());
+        }
+
+        return true;
+    }
+
+    static bool HandleResetItemsInVendorBuyBackTabCommand(ChatHandler* handler, Optional<PlayerIdentifier> target)
+    {
+        Player* targetPlayer = GetPlayerFromIdentifierOrSelectedTarget(handler, target);
+
+        if (!targetPlayer)
+        {
+            return false;
+        }
+        else
+        {
+            int16 deletedItemsCount = ResetItemsInVendorBuyBackTab(targetPlayer);
+            handler->PSendSysMessage(LANG_COMMAND_RESET_ITEMS_BUYBACK, deletedItemsCount, handler->GetNameLink(targetPlayer).c_str());
+        }
+
+        return true;
+    }
+
+    static bool HandleResetItemsAllCommand(ChatHandler* handler, Optional<PlayerIdentifier> target)
+    {
+        Player* targetPlayer = GetPlayerFromIdentifierOrSelectedTarget(handler, target);
+
+        if (!targetPlayer)
+        {
+            return false;
+        }
+        else
+        {
+
+            // Delete all items destinations
+            int16 deletedItemsEquipedCount            = ResetItemsEquipped(targetPlayer);
```

Comment:
```suggestion
            int16 deletedItemsEquippedCount           = ResetItemsEquipped(targetPlayer);
```
Three more instances in this file.

Reply:
@heyitsbench 
I have fixed it on the last commit.
I don't know why they didn't appear in my last searches for replacements O_o.